oak/server/rust/oak_runtime/src/runtime/tests.rs

```diff
@@ -140,3 +140,68 @@ fn create_node_more_public_label() {
         }),
     );
 }
+
+#[test]
+fn wait_on_channels_immediately_returns_if_any_channel_is_orphaned() {
+    run_node_body(
+        Label::public_trusted(),
+        Box::new(|runtime| {
+            // Create channels with handles
+            let (write_handle, read_handle_1) = runtime.channel_create(&Label::public_trusted());
```

Comment:
```suggestion
            let (write_handle_1, read_handle_1) = runtime.channel_create(&Label::public_trusted());
```

for consistency (and `_2` below).

Also in other places we start from 0, but up to you really.